src/terra-dev-site/contributing/DeveloperWorkflow.c.contributing.md

```diff
@@ -0,0 +1,171 @@
+# Development Workflow
+
+## Create Tech Design
+It's good to create a tech design on the related GitHub issue you'll be working on. This allows for discussion with the [terra team](https://github.com/orgs/cerner/teams/terra) before getting deep into development. Once you feel the tech design has been landed on, feel free to open up a pull request. Based on the changes introduced in the PR, the change will either be considered a small or large change and follow review guidelines documented in the [terra governance modal](https://github.com/cerner/terra-ui/blob/master/GOVERNANCE.md).
+
+Based on feedback you receive from tech design review, incorporate it into your implementation. It's helpful to update the tech design with feedback you incorporate.
+
+## Tech Design Format
+
+```
+# Tech Design
+## Expected behavior
+/* Describe expected behavior, include visuals if helpful */
+
+## Code Sample
+/* Describe proposed code change at high-level */
+
+/* Include sample code if applicable */
+
+## Props
+/* Describe react props */
+
+| Prop Name | Type | Is Required | Default Value | Description |
+|-|-|-|-|-|
+| text |string | required | none | Sets the button text. |
+| href |string | optional | none | Sets the href. When set will render the component as an anchor tag. |
+
+## Responsiveness
+/* Describe intended responsive behavior if applicable */
+
+## Internationalization
+/* Describe intended internationalization behavior if applicable */
+
+## Accessibility
+/* Describe intended accessible behavior if applicable */
+```
+
+## Starting Development
+The following guide covers the development workflow for the terra monorepos that are managed with [Lerna](https://lernajs.io/) to handle shared and cross-package dependencies.
+
+* [terra-core](https://github.com/cerner/terra-core)
+* [terra-framework](https://github.com/cerner/terra-framework)
+* [terra-clinical](https://github.com/cerner/terra-clinical)
+
+Components along with their docs and tests are organized as individual packages that share similar tooling. You can read more about the [directory structure of these components here](https://github.com/cerner/generator-terra-module#generated-package).
+
+## Install Dependencies
+
+After you’ve gone through the [git workflow](#/contributing/terra-ui/git-workflow) and created your branch, the next step will be to install the project dependencies.
+
+Run the following command to install the npm dependencies.
+
+`npm install`
+
+Make sure you’re using the correct NodeJS version (we recommend using [nvm](https://github.com/creationix/nvm) to install and manage node on your machine)
```

Comment:
We could link back to this for NVM / NPM: https://github.com/cerner/terra-ui/pull/108/files#diff-747f03c691438a04d3fa074e8c066a12R46

Reply:
I'm gonna drop this bit, forgot it was already documented in the prereqs section.